cms-derived-data/config.yaml

```diff
@@ -0,0 +1,50 @@
+---
+# all recids are placeholders, should be changed later to correct ones
+common_values:
+  abstract:
+    description: <p>Dataset in reduced NanoAOD format for education and outreach derived from the primary dataset in AOD format linked below.</p>
```

Comment:
POET, Run1 and PF will have different descriptions with a common final part which will be "derived from the primary dataset in MiniAOD format linked below."